docs/release-process.md

```diff
@@ -8,73 +24,38 @@
  2. Package two days before a release. So if the release is to be on Thursday,
     package on Tuesday. Think about timezones as well.
 
- 3. Ensure that the tests on Travis CI are green.
+ 3. Ensure that the relevant tests on CI are green.
 
-    See: https://travis-ci.org/php/php-src/builds
+    See: 
+      https://travis-ci.org/php/php-src/builds
+      https://dev.azure.com/phpazuredevops/PHP/
 
     It is recommended to do so a couple of days before the packaging day, to
     have enough time to investigate failures, communicate with the authors and
     commit the fixes.
 
-    The RM for the branch is also responsible for keeping the CI green on
-    ongoing basis between the releases. Check the CI status for your branch
-    periodically and resolve the failures ASAP. See more in
-    https://wiki.php.net/rfc/travis_ci.
+    Check the CI status for your branch periodically and resolve the failures ASAP. 
+    
+    See more in https://wiki.php.net/rfc/travis_ci.
 
- 4. Ensure that Windows builds will work before packaging.
-
- 5. Follow all steps to the letter. When unclear ask previous RM's (David,
-    Julien, Johannes, Stas, Derick, Ilia, Sara, Remi, or Christoph) before
+ 4. Follow all steps to the letter. When unclear ask previous RM's before
     proceeding. Ideally make sure that for the first releases one of the
     previous RM's is around to answer questions. For the steps related to the
     php/QA/bug websites try to have someone from the webmaster team (Bjori) on
     hand.
 
- 6. Verify the tags to be extra sure everything was tagged properly.
-
- 7. Moving extensions from/to PECL requires write access to the destination.
-    Most developers should have this.
-
-    Moving extensions from php-src to PECL:
```

Comment:
I agree that moving extension from/to PECL isn't the job of RMs, but the info appears to be generally useful. Maybe move that to the Wiki? Where exactly?

Reply:
The intention is to just move it out of this document, a wiki page sounds good to me. I'm not sure where we would link to the wiki page.

Whatever this section is unrelated to release management.